modules/oe_theme_content_publication/oe_theme_content_publication.post_update.php

```diff
@@ -65,10 +66,49 @@ function oe_theme_content_publication_post_update_00004(): void {
   $display->save();
 }
 
+/**
+ * Enable Field Group and OpenEuropa Theme Content Entity Contact modules.
+ */
+function oe_theme_content_publication_post_update_00005(): void {
+  \Drupal::service('module_installer')->install(['oe_theme_content_entity_contact']);
+}
+
+/**
+ * Add Publication thumbnail image style.
+ */
+function oe_theme_content_publication_post_update_00006() {
+  // If the image style already exists, we bail out.
+  $style = \Drupal::entityTypeManager()->getStorage('image_style')->load('oe_theme_publication_thumbnail');
+  if ($style) {
+    return 'The image style was previously created.';
+  }
+
+  // Create image style.
+  $image_style = ImageStyle::create([
+    'name' => 'oe_theme_publication_thumbnail',
+    'label' => 'Publication thumbnail',
+  ]);
+
+  // Create effect.
+  $effect = [
+    'id' => 'image_scale',
+    'weight' => 1,
+    'data' => [
+      'width' => 192,
+      'height' => 192,
+      'upscale' => FALSE,
+    ],
+  ];
+
+  // Add effect to the image style and save.
+  $image_style->addImageEffect($effect);
+  $image_style->save();
+}
+
 /**
  * Create the 'full' entity view display on the publication CT.
  */
-function oe_theme_content_publication_post_update_00005() {
+function oe_theme_content_publication_post_update_00007() {
```

Comment:
When running an upgrade path from a dump from 2.x, I get:

![image](https://user-images.githubusercontent.com/153362/100217422-b9061600-2f13-11eb-9534-68c08c8cf452.png)

Reply:
I've fixed it. But if we want to test update correctly need to update it from tag 2.11 because I've merged changes from EWPP-354 with new changes from this ticket. EWPP-354 is in the 2.x already, so 2.x -> EPIC-Publication won't be updated correctly (full view won't be updated).

Reply:
Understood, it's correct, I should use a database from a stable version and not from a 2.x. Or if I do it it's at my own risk. All good then, resolving.